app/views/backdate/edit.html.erb

```diff
@@ -0,0 +1,50 @@
+<% content_for :back_link, render_back_link(href: document_path(@edition.document)) %>
+<% content_for :title, t("backdate.edit.title") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render_govspeak(t("backdate.edit.intro_paragraph")) %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% legend = capture do %>
+      <span class="govuk-heading-s govuk-!-margin-bottom-0">
+        <%= t("backdate.edit.date") %>
+      </span>
+    <% end %>
+    <% date = @edition.backdated_to %>
+
+    <%= form_tag backdate_path(@edition.document) do %>
+      <%= render "govuk_publishing_components/components/date_input", {
+        legend_text: legend,
+        hint: t("backdate.edit.date_hint_text"),
+        name: "backdate",
```

Comment:
I think this would be more consistent with other forms if it was backdate[date], as these day, month and year are just properties of this date field.

Reply:
Sure, I figured since we were only dealing with the date on this page it didn't need additional labelling.

Reply:
Yup, you're right it doesn't actually need it, but does slip into inconsistency without it.